openj9.test.jlm/src/test.jlm.ibm/net/openj9/test/management/TestOperatingSystemMXBean.java

```diff
@@ -1,158 +1,158 @@
+	 */
+	public  void testGetProcessCpuLoad() throws MalformedObjectNameException, IOException {
+		int i = 0;
+		double processLoad = 0;
+
+		// First call to getProcessCpuLoad() should have returned -1 
+		assertEquals(-1.0, processLoadFirstCall);
+
+		// Generate some load before reading up Process CPU Time yet again.
+		try {
+			Thread[] busyObj = new Thread[NTHREADS];
+			long counter = 0;
+
+			for (; counter < NITERATIONS; counter++) {
+				for (i = 0; i < NTHREADS; i++) {
+					busyObj[i] = new Thread(new BusyThread( SLEEPINTERVAL * 500));
+					busyObj[i].start();
+				}
+				Thread.sleep(1000);
+				for (i = 0; i < NTHREADS; i++) {
+					busyObj[i].join();
+				}
+			}
+		} catch(InterruptedException e) {
+			fail("Test Failed" + e);
+		}
+
+		// Now obtain the process's CPU load; ensure that load has increased. 
+		processLoad = osmxbean.getProcessCpuLoad();
+
+		// Process CPU Load should have increased and hence, in the [0.0, 1.0] range.
+		if (processLoad >= 0 && processLoad <= 1.0 ) {
+			// Do nothing
+		} else {
+			fail("processLoad should have been between 0 to 1.0 but " + "the value is: " + processLoad);
+		}
+	}
+
+	/**
+	 * Test the getSystemCpuLoad() API of OperatingSystemMXBean
+	 */
+	public void testGetSystemCpuLoad() throws MalformedObjectNameException, IOException {
+		/* The API is unsupported on z/OS; remove this test and return
+		 * once support is available. */
+		if ((null != OSNAME) && (OSNAME.equalsIgnoreCase("z/OS"))) { 
+			// Check for unsupported return code and just return
+			assertEquals(-3.0, systemCpuLoadFirstCall);
+			return;
+		}
+
+		// First call to getSystemCpuLoad() should have returned -1 
+		assertEquals(-1.0, systemCpuLoadFirstCall);
+
+		// Perform some CPU intensive work to ensure CPU cycles have 
+		// been spent before obtaining CPU load.
+		try {
+			RandomNumber random = new RandomNumber();
+
+			long limit = random.generateRandomNumber(1, Long.MAX_VALUE);
+
+			// Form fibonacci series until the limit is reached.
+			Thread thread1 = new Thread(new FibonacciSeriesOSMXBean(limit));
+			thread1.start();
+
+			thread1.join();
+			System.out.println(SHARED_RESULT);
+
+			// Measuring the system CPU load after starting the work load.
+			double systemCpuLoad = osmxbean.getSystemCpuLoad();
+
+			assertTrue("SystemCpuLoad should have been between 0.0 " + "to 1.0 but the value is " + systemCpuLoad, 
+					(systemCpuLoad >= 0 && systemCpuLoad <= 1.0));
+
+		} catch(InterruptedException ie) {
+			fail("Test Failed" + ie);
+		}
+	}
+}
+
+class FibonacciSeriesOSMXBean implements Runnable {
+	long limit;
+	long result;
+
+	public FibonacciSeriesOSMXBean(long limit) {
+		this.limit = limit;
+	}
+
+	public void run() {
+
+		long start = 1;
+		long next = 1;
+		long result = start;
+
+		while( result < (limit*0.75) ) {
+			long output = start + next;
+			result = output;			
+			start = next;
+			next = output;
+		}
+
+		TestOperatingSystemMXBean.SHARED_RESULT = result;
+	}
+}
```

Comment:
A number of the test.jlm files had Windows CRLF rather than unix line feed so I fixed them - but that caused the PR checker to insist on updating the copyright year.